### PR TITLE
New package: gnome-screensaver-3.6.1

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: srcpkgs/gnome-screensaver/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'gnome-screensaver'
+pkgname=gnome-screensaver
+version=3.6.1
+revision=1
+build_style=gnu-configure
+configure_args="--enable-compile-warnings=minimum"
+hostmakedepends="pkg-config automake glib-devel"
+makedepends="gnome-desktop-devel dbus-glib-devel pam-devel intltool"
```
Comment (maxice8) [inline srcpkgs/gnome-screensaver/template]: intltool should be in hostmakedepends

Comment (shizonic) [reply srcpkgs/gnome-screensaver/template]: Will fix this asap.

Comment (maxice8) [reply srcpkgs/gnome-screensaver/template]: once that change is done check if lines 17 to 19 are still of any use

Comment (shizonic) [reply srcpkgs/gnome-screensaver/template]: @maxice8 Ok. Will do that. Thanks.

**Comment by CameronNemo** [comment]
This is unmaintained. Any reason to use it rather than an alternative such as light locker or GDM's lock screen?

**Comment by shizonic** [comment]
@maxice8 Done like you suggested. Now it build on every arch.
@CameronNemo Yes because it's used for the `gnome-flashback-metacity` session.

**Pull request merged**